plugin/pulp/plugin/__init__.py

```diff
@@ -1,4 +1,5 @@
 from pulp.app.models import ProgressBar, ProgressSpinner, Repository, RepositoryContent
+from pulp.exceptions import PulpException, MissingResource
```

Comment:
I don't think the plugin api needs MissingResource. I had imagined that all url parameter validation for the view layer will happen in the platform views. Also we need to have an issue for how to handle missing resource, but that is a separate matter.